lib/private/AppConfig.php

```diff
@@ -1176,29 +1176,30 @@ private function assertParams(string $app = '', string $configKey = '', bool $al
 		}
 	}
 
-	private function loadConfigAll(): void {
-		$this->loadConfig(null);
+	private function loadConfigAll(?string $app = null): void {
+		$this->loadConfig($app, null);
 	}
 
 	/**
 	 * Load normal config or config set as lazy loaded
 	 *
 	 * @param bool|null $lazy set to TRUE to load config set as lazy loaded, set to NULL to load all config
 	 */
-	private function loadConfig(?bool $lazy = false): void {
+	private function loadConfig(?string $app = null, ?bool $lazy = false): void {
 		if ($this->isLoaded($lazy)) {
 			return;
 		}
 
-		if (($lazy ?? true) !== false) { // if lazy is null or true, we debug log
-			$this->logger->debug('The loading of lazy AppConfig values have been requested', ['exception' => new \RuntimeException('ignorable exception')]);
+		// if lazy is null or true, we debug log
+		if (($lazy ?? true) !== false && $app !== null) {
+			$this->logger->debug('The loading of lazy AppConfig values have been triggered by app "' . $app . '"', ['exception' => new \RuntimeException($app)]);
```

Comment:
```suggestion
			$this->logger->debug('The loading of lazy AppConfig values have been triggered by app "' . $app . '"', ['exception' => new \RuntimeException('The loading of lazy AppConfig values have been triggered by app "' . $app . '"'), 'app' => $app]);
```

1. Pass a full message into the exception for tools like Sentry, where the exception message can be seen more prominently than the log message
2. Set the app context

Reply:
Makes sense